modules/network/private-endpoint/main.bicep

```diff
@@ -18,7 +18,7 @@ param applicationSecurityGroupResourceIds array?
 param customNetworkInterfaceName string?
```

Comment:
The challenge with these updates will be that we'll need to port them to AVM asap. Will you do that, or should somebody else (once approved)?

Reply:
Also, we'd need to update the [AVM specs](https://azure.github.io/Azure-Verified-Modules/specs/shared/interfaces/#private-endpoints)

Reply:
Don't mind doing that, will reach out offline